planet/scripts/cli.py

```diff
@@ -103,3 +107,37 @@ def init(email, password):
     response = call_and_wrap(clientv1().login, email, password)
     write_planet_json({'key': response['api_key']})
     click.echo('initialized')
+
+@cli.command('quota')
+def quota():
+    '''Print allocation and remaining quota in Sqkm.'''
+    try:
+        with open (os.path.join(os.path.expanduser('~'), '.planet.json')) as f:
+            data=json.load(f)
+        main=requests.get('https://api.planet.com/auth/v1/experimental/public/my/subscriptions', auth=HTTPBasicAuth(data["key"], ''))
+        if main.status_code==200:
+            content=main.json()
+            n=(str(main.json()).count("quota_sqkm"))
+            i=0
+            while i<n:
+                item_id=(content[i])
+                print(" ")
```

Comment:
use `click.echo` instead of `print()`, for consistency with the rest of this file (see also: http://click.pocoo.org/5/utils/#printing-to-stdout)

Reply:
Resolved